read-docs/start.sh

```diff
@@ -5,4 +5,4 @@ set -a
 . $DIR/../.env
 set +a
 
-npx babel-watch ${DIR}/index.js
```

Comment:
@satrams was this change intentional?

Reply:
Haha no I must've forgotten to change it back